test/fetch_multi_test.rb

```diff
@@ -265,6 +265,12 @@ def test_fetch_multi_with_open_transactions_returns_non_readonly_records
     end
   end
 
+  def test_fetch_multi_with_no_keys_does_not_query_when_cache_is_disabled
+    Item.stubs(:should_use_cache?).returns(false)
+    Item.expects(:where).never
```

Comment:
never expectations are extra brittle.  I would prefer if `assert_queries(0)` and `assert_memcache_operations(0)` were used